src/lib/gl_engine/meson.build

```diff
@@ -19,9 +19,9 @@ source_file = [
    'tvgGlShaderSrc.cpp',
 ]
 
-egl_dep = meson.get_compiler('cpp').find_library('EGL')
 gles_dep = meson.get_compiler('cpp').find_library('GLESv2')
```

Comment:
Should `gles_dep` also be updated to `GLESv3`? Tried building on Android, got error `undefined reference to `glGenVertexArrays'`, upgrading library here fixed a problem.